torchci/lib/bot/retryBot.ts

```diff
@@ -10,22 +10,32 @@ function retryBot(app: Probot): void {
       ctx.payload.workflow_run.conclusion === "success" ||
       ctx.payload.workflow_run.head_branch !== "master" ||
       attemptNumber > 1 ||
-      !["lint", "pull"].includes(workflowName)  // only do lint and pull for now because they are fast
+      !["Lint", "pull"].includes(workflowName) // only do lint and pull for now because they are fast
```

Comment:
alt idea, make it case insensitive
```suggestion
      !["lint", "pull"].includes(workflowName.toLowerCase()) // only do lint and pull for now because they are fast
```